src/plugins/bfetch/public/batching/create_streaming_batched_function.ts

```diff
@@ -88,6 +88,10 @@ export const createStreamingBatchedFunction = <Payload, Result extends object>(
           return !item.signal?.aborted;
         });
 
+        if (items.length === 0) {
+          return; // all items have been aborted before a request has been sent
```

Comment:
just small optimization: fixing the case when Kibana could send batch request with empty batch array